types/atom/src/grammar-registry.d.ts

```diff
@@ -1,5 +1,12 @@
 import { Disposable, Grammar, GrammarToken, TextBuffer } from '../index';
 
+type SyntaxNode = any;
```

Comment:
CI complains about this, and I really don't like this either, because `any` opens a gaping hole in the type system, essentially, anything with `any` isn't typechecked. 

I think we can do better: define an empty `interface SyntaxNode`. Consumers then can extend it in case they need to do something with syntax nodes beyond passing them around.